backend/server/models.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from peewee import *
+
+if os.environ["ENV"] == "test":
+    db = SqliteDatabase(":memory:")
+else:
+    db = SqliteDatabase("sqlite.db")
+
```

Comment:
`ENV` が宣言されてないと転ぶので getを使いたいです

```suggestion
if  os.environ.get("ENV") == "test":
    db = SqliteDatabase(":memory:")
else:
    db = SqliteDatabase("sqlite.db")
```